### PR TITLE
Removes the code/unused folder.

### PR DESCRIPTION
**Pull request by PJB3005**
Also axes the supermatter code under cael's WiP folder.

Hey did you know that zero point lasers (whatever those were) were subtypes of computers?

### CODE REVIEW

**Comment by clusterfack** [comment]
Not sure about this one, pretty sure some of this shit is used

**Comment by PJB3005** [comment]
@clusterfack none is, if you look at the dme none of the files in code/unused were included, travis passed, and for the cael SM stuff, well it compiled even by simply killing the files.

**Comment by clusterfack** [comment]
It'll compile even if you remove all the sprites and make things invisible

**Comment by PJB3005** [comment]
@clusterfack no it won't it'll say missing file.

Unless somebody referenced a file with " instead of ' but that will deserve slapping.

File: code/unused/goonheist/genetics2/geneticsMachines.dm [hunk 1]
```diff
@@ -1,974 +0,0 @@
-var/list/genetics_computers = list()
```
Comment (Iamgoofball) [inline code/unused/goonheist/genetics2/geneticsMachines.dm]: Keep the goonheist folder. This is goonetics, and should be preserved for historical purposes, even if it isn't being used.

Comment (PJB3005) [inline code/unused/goonheist/genetics2/geneticsMachines.dm]: If people want to go into the git history, they can.

Comment (Iamgoofball) [inline code/unused/goonheist/genetics2/geneticsMachines.dm]: Git history is bad at pulling back entire folders.

Comment (PJB3005) [inline code/unused/goonheist/genetics2/geneticsMachines.dm]: no it's not.

**Comment by DeityLink** [comment]
I don't like this. I like browsing through unused code for unfinished ideas and ponder if I should give them a second life.

The prehistoric plants are such an example.

**Pull request closed without merging**